### PR TITLE
Load feed

### LINKED ISSUE #48: Load feed
We need to load the sessions into the app. 

API Path: `events/droidconke-2022-281/feeds`

Design Reference: https://xd.adobe.com/view/dd5d0245-b92b-4678-9d4a-48b3a6f48191-880e/screen/aabbe326-f4d9-4fae-9cbc-433491567613

Steps
Similar to the guidelines from https://github.com/droidconKE/flutterconKEApp/issues/44

Notes
- If you run into any problem, you can add a comment to this issue to have it resolved

### PR DESCRIPTION
**Pull request by DevshiDeno**
## Description
load feed issue

I created the necessary models required to handle event feeds gotten from the API and used Cubit to track state on the feed UI screen.

Closes #48 

## Type of Change
enhancement

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [x] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

## Test Changes on Device
You can download your APK from Firebase App Distribution once this PR builds successfully via the following link: https://appdistribution.firebase.dev/i/c796669942f8a811

### CODE REVIEW

**Comment by MillerAdulu** [comment]
@DevshiDeno 

Kindly add a video description to showcase the interaction of the UI concerning this feature.

File: lib/features/feed/cubit/feed_cubit.dart [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  }) : super(const FetchFeedState.initial()) {
+    _apiRepository = apiRepository;
+  }
+
+  late ApiRepository _apiRepository;
+
+  Future<void> fetchFeeds() async {
+    emit(const FetchFeedState.loading());
+    try {
+      final feeds = await _apiRepository.fetchFeeds(event: '');
```
Comment (MillerAdulu) [inline lib/features/feed/cubit/feed_cubit.dart:22]: The slug for the feed of this event as per the documentation at this URL should be `droidconke-2022-281`

https://documenter.getpostman.com/view/3385291/SzS4TTXb#a443ec01-1cf8-41d9-bfd5-56fb6c228de4

Please make this slight addition.

**Comment by DevshiDeno** [comment]
Okay

File: linux/flutter/ephemeral/.plugin_symlinks/path_provider_linux
Comment (MillerAdulu) [inline linux/flutter/ephemeral/.plugin_symlinks/path_provider_linux:1]: Delete this file.

File: linux/flutter/generated_plugin_registrant.cc
Comment (MillerAdulu) [inline linux/flutter/generated_plugin_registrant.cc:1]: Delete this file.

File: linux/flutter/generated_plugin_registrant.h
Comment (MillerAdulu) [inline linux/flutter/generated_plugin_registrant.h:1]: Delete this file.

File: linux/flutter/generated_plugins.cmake
Comment (MillerAdulu) [inline linux/flutter/generated_plugins.cmake:1]: Delete this file.

File: macos/Flutter/GeneratedPluginRegistrant.swift
Comment (MillerAdulu) [inline macos/Flutter/GeneratedPluginRegistrant.swift:1]: Delete this file.

File: macos/Flutter/ephemeral/Flutter-Generated.xcconfig
Comment (MillerAdulu) [inline macos/Flutter/ephemeral/Flutter-Generated.xcconfig:1]: Delete this file.

File: macos/Flutter/ephemeral/flutter_export_environment.sh
Comment (MillerAdulu) [inline macos/Flutter/ephemeral/flutter_export_environment.sh:1]: Delete this file.

**Review by MillerAdulu — CHANGES_REQUESTED** [review]

**Comment by MillerAdulu** [comment]
@DevshiDeno,

Once you are done with the changes, check the failing pipeline here: https://github.com/droidconKE/flutterconKEApp/actions/runs/10305939605/job/28533957201?pr=54

Fix the formatting issue by running the `fmt` command in the `Makefile`

**Comment by MillerAdulu** [comment]
The code generally looks OK. Looking forward to the corrections and this can be merged in quickly.

**Comment by MillerAdulu** [comment]
@DevshiDeno,

Trusting that you are still on this.

**Comment by DevshiDeno** [comment]
Yes, I am working on it.

**Comment by MillerAdulu** [comment]
@rebase

**Comment by MillerAdulu** [comment]
Made changes on #59 that's now merged. Closing

**Pull request closed without merging**